contrib/cirrus/lib.sh

```diff
@@ -449,63 +448,43 @@ systemd_banish() {
     $GOSRC/$PACKER_BASE/systemd_banish.sh
 }
 
-install_buildah_packages() {
-    git clone https://github.com/containers/buildah.git /tmp/buildah
-    if [[ -r "$BUILDAH_PACKAGES_FILEPATH" ]]; then
-        source "$BUILDAH_PACKAGES_FILEPATH"
-        req_env_var UBUNTU_BUILDAH_PACKAGES FEDORA_BUILDAH_PACKAGES OS_RELEASE_ID
-        case "$OS_RELEASE_ID" in
-            fedora)
-                $BIGTO ooe.sh sudo dnf install -y ${FEDORA_BUILDAH_PACKAGES[@]}
-                ;;
-            ubuntu)
-                $LILTO $SUDOAPTGET update
-                $BIGTO $SUDOAPTGET install ${UBUNTU_BUILDAH_PACKAGES[@]}
-                ;;
-            *) bad_os_id_ver ;;
-        esac
-    else
-        warn "Could not find $BUILDAH_PACKAGES_FILEPATH in buildah repository root."
-    fi
-}
-
 _finalize() {
     set +e  # Don't fail at the very end
     if [[ -d "$CUSTOM_CLOUD_CONFIG_DEFAULTS" ]]
     then
         echo "Installing custom cloud-init defaults"
-        sudo cp -v "$CUSTOM_CLOUD_CONFIG_DEFAULTS"/* /etc/cloud/cloud.cfg.d/
+        $SUDO cp -v "$CUSTOM_CLOUD_CONFIG_DEFAULTS"/* /etc/cloud/cloud.cfg.d/
     else
         echo "Could not find any files in $CUSTOM_CLOUD_CONFIG_DEFAULTS"
     fi
     echo "Re-initializing so next boot does 'first-boot' setup again."
     cd /
-    sudo rm -rf /var/lib/cloud/instanc*
-    sudo rm -rf /root/.ssh/*
-    sudo rm -rf /etc/ssh/*key*
-    sudo rm -rf /etc/ssh/moduli
-    sudo rm -rf /home/*
-    sudo rm -rf /tmp/*
-    sudo rm -rf /tmp/.??*
-    sudo sync
-    sudo fstrim -av
+    $SUDO rm -rf /var/lib/cloud/instanc*
+    $SUDO rm -rf /root/.ssh/*
+    $SUDO rm -rf /etc/ssh/*key*
+    $SUDO rm -rf /etc/ssh/moduli
+    $SUDO rm -rf /home/*
+    $SUDO rm -rf /tmp/*
+    $SUDO rm -rf /tmp/.??*
+    $SUDO sync
+    $SUDO fstrim -av
 }
 
 rh_finalize() {
     set +e  # Don't fail at the very end
     echo "Resetting to fresh-state for usage as cloud-image."
     PKG=$(type -P dnf || type -P yum || echo "")
-    sudo $PKG clean all
-    sudo rm -rf /var/cache/{yum,dnf}
-    sudo rm -f /etc/udev/rules.d/*-persistent-*.rules
-    sudo touch /.unconfigured  # force firstboot to run
+    $SUDO $PKG clean all
+    $SUDO rm -rf /var/cache/{yum,dnf}
+    $SUDO rm -f /etc/udev/rules.d/*-persistent-*.rules
+    $SUDO touch /.unconfigured  # force firstboot to run
     _finalize
 }
 
 ubuntu_finalize() {
     set +e  # Don't fail at the very end
     echo "Resetting to fresh-state for usage as cloud-image."
     $LILTO $SUDOAPTGET autoremove
-    sudo rm -rf /var/cache/apt
+    $SUDO rm -rf /var/cache/apt
```

Comment:
```suggestion
    $SUDO rm -r /var/cache/apt
```
If you expect the cache to really be empty drop the -f.

Reply:
Not sure I follow the suggestion.  This is another VM Image build-time thing where it's probably okay to ignore a 'does not exist' error.  Am I missing something?